src/meltano/core/setting_definition.py

```diff
@@ -170,7 +170,6 @@ def __init__(
         options: list | None = None,
         oauth: dict | None = None,
         placeholder: str | None = None,
-        protected: bool | None = None,
```

Comment:
There are plugins in the Hub that still use `protected: true`, should we keep it until we update those?

FWIW, there seem to be only 2 plugins using it: https://github.com/search?q=repo%3Ameltano%2Fhub%20protected&type=code

Reply:
Good callout. The setting no longer does anything (and has a confusing name), so I'm tempted to remove it while we're making this already-breaking change.

Do you think it'd be safe to remove that setting from those plugins on the Hub ahead of the Meltano 3.0 release?

fyi @pnadolny13

Reply:
> Do you think it'd be safe to remove that setting from those plugins on the Hub ahead of the Meltano 3.0 release?

Yes, I think so.

Reply:
Started https://github.com/meltano/hub/pull/1271 to get the ball rolling on the Hub side